upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -276,16 +276,11 @@ func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval
 	if force {
 		update = append(update, n.Ready...)
 	}
+
 	for _, u := range update {
 		instanceID := aws.StringValue(u.ASGInstance.InstanceId)
 		glog.Infof("Stopping instance %q in AWS ASG %q", instanceID, n.ASGName)
 
-		// TODO: Evacuate through k8s first?
-
-		// TODO: Temporarily increase size of ASG?
```

Comment:
let's leave these comments - I think the only addresses the "Evacuate through k8s first" comment